src/services/bags.ts

```diff
@@ -97,7 +98,7 @@ export async function bagsListCheck(api: ApiPromise, account: KeyringPair, sendT
 
 	const txsInner = needRebag.map((who) => api.tx.bagsList.rebag(who)).slice(0, count);
 	const tx = api.tx.utility.batchAll(txsInner);
-	const [success, _] = await dryRun(api, account, tx);
```

Comment:
Do you know if there something like `_` in rust in TS? I want to bind a variable, but I don't want to use it, and I don't want complains about it.

Reply:
also, is this the same code now? as in, `success` stays the same before and after diff?

Reply:
Success stays the same. Why you want the 2nd variable to be bound since you wont use it?

Reply:
Presumably just to be explicit, but I don't think its critical.

Also, I think you can write a custom rule to get the linter to ignore variables that start with `_*`, but I have always been surprised it doesn't do this out of the box

Reply:
Well yes - but in our world (JS/TS) it is strange :D I can see the point though of binding from a more normal language than JS